code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -2414,3 +2413,122 @@ datum
 			result = "rewriter"
 			required_reagents = list("spacemountainwind" = 1, "coffee" = 1)
 			result_amount = 2
+
+		vinegar
+			name = "Vinegar"
+			id = "vinegar"
+			result = "vinegar"
+			required_reagents = list("wine" = 5)
+			required_catalysts = list("enzyme" = 5)
+			result_amount = 5
+
+
+
+//cafe stuff!
+
+
+		acidtea
+			name = "Earl's Grey Tea"
+			id = "acidtea"
+			result = "acidtea"
+			required_reagents = list("sacid" = 5, "tea" = 5)
+			result_amount = 10
+
+		chifir
+			name = "Chifir"
+			id = "chifir"
+			result = "chifir"
+			required_reagents = list("tea" = 5, "redtea" = 5, "greentea" = 5)
+			result_amount = 15
+
+		yinyang
+			name = "Zen Tea"
+			id = "yinyang"
+			result = "yinyang"
+			required_reagents = list("tea" = 5, "nothing" = 5)
+			result_amount = 10
+
+		singularitea
+			name = "Singularitea"
+			id = "singularitea"
+			result = "singularitea"
+			required_reagents = list("radium" = 1, "tea" = 5, "redtea" = 5)
+			result_amount = 10
+
+		gyro
+			name = "Gyro"
+			id = "gyro"
+			result = "gyro"
+			required_reagents = list("greentea" = 5, "whiskey" = 5, "iron" = 1)
+			result_amount = 10
+
+		plasma
+			name = "Plasma Pekoe"
+			id = "plasma"
+			result = "plasma"
+			required_reagents = list("tea" = 5, "plasma" = 5)
+			result_amount = 10
+
+		espresso
+			name = "Espresso"
+			id = "espresso"
+			result = "espresso"
+			required_reagents = list("coffee" = 1, "water" = 1)
+			result_amount = 2
+
+
+		cappuccino
+			name = "Cappuccino"
+			id = "cappuccino"
+			result = "cappuccino"
+			required_reagents = list("espresso" = 1, "cream" = 1)
+			result_amount = 2
+
+		tonio

```

Comment:
are you fucking kidding me

this is literally doctor's delight: improved edition

fuck off with this shit